homeassistant/components/climate/systemair_savecair.py

```diff
@@ -0,0 +1,282 @@
+"""
+Support for Systemair Savecair climate devices.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/climate.systemair_savecair/
+"""
+import voluptuous as vol
+
+from homeassistant.components.climate import (
+    PLATFORM_SCHEMA, STATE_AUTO, STATE_IDLE, STATE_MANUAL, SUPPORT_AWAY_MODE,
+    SUPPORT_FAN_MODE, SUPPORT_ON_OFF, SUPPORT_OPERATION_MODE,
+    SUPPORT_TARGET_TEMPERATURE, ClimateDevice)
+from homeassistant.const import (
+    ATTR_TEMPERATURE, CONF_NAME, CONF_PASSWORD, TEMP_CELSIUS, TEMP_FAHRENHEIT)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['python-systemair-savecair==1.0.1']
```

Comment:
This release is broken since the inner systemair package in the source lacks a `__init__.py` file so the directory isn't identified as a python packge.